### ISSUE TITLE
Library function available in all docs?

### ISSUE THREAD

**Issue by nomicode** [mangini/gdocs2md#24]
Is there a way to save a library function or script that is available in all docs. I don't want to have to go through this whole process for each doc.

**Comment by lpanebr** [comment]
Perhaps we could try to make it into an add-on.

**Comment by evbacher** [comment]
I agree. I'm going to try and make it an add-on (and maybe some other improvements too).

**Comment by nomicode** [comment]
Cool, thanks!

**Comment by wyonghao** [comment]
Hello, I am too like to have it for all my google docs? I searched some keywords in google doc add-on store. There is no markdown related tools. Please let us know how to make the script easily to be accessed for all docs? Many thanks!
